### DISCUSSION TITLE
(REQUEST) Cow Mangler Charged Projectile & Spy Auto Zapper.

### DISCUSSION THREAD

**Discussion by yujispin990** [Fedoraware/Fedoraware#972] [Ideas]
Just like the title implies an aimbot that accounts for the cow mangler charged shot and well...auto zapper.

**Comment by Ragdoll117** [comment]
Auto Sapper already exists within the aimbot or something

**Comment by yujispin990** [comment]
wait really? i didnt notice, do i need to keep pressing the button for it to activate because i was hoping for something like what lbox has where you just pull it out and it does the rest for you.

**Comment by yujispin990** [comment]
the charged mangler though would slap hard bro combine with the already impressive prediction.

**Comment by ghost** [comment]
charged mangler shot has not yet been added.

**Comment by yujispin990** [comment]
Oh ok, thank you for reopening!
